src/wallet/walletdb.cpp

```diff
@@ -1114,12 +1063,99 @@ static DBErrors LoadAddressBookRecords(CWallet* pwallet, DatabaseBatch& batch) E
     return DBErrors::LOAD_OK;
 }
 
+static DBErrors LoadTxRecords(CWallet* pwallet, DatabaseBatch& batch, std::vector<uint256> upgraded_txs, bool& any_unordered) EXCLUSIVE_LOCKS_REQUIRED(pwallet->cs_wallet)
```

Comment:
In 890143f2:

`upgraded_txs` is a return arg, needs to be passed by reference.

Reply:
Fixed